### DISCUSSION TITLE
公式与正文间距问题

### DISCUSSION THREAD

**Discussion by eutopia-61** [note286/xduts#63] [Q&A] [answered]
我看word模板要求公式与正文上下都空一行，不知道是不是我的设置问题
![image](https://user-images.githubusercontent.com/67130065/224327489-a8cec9da-46c4-41a3-8293-e229389c3529.png)
![image](https://user-images.githubusercontent.com/67130065/224327894-53f00765-ffe3-41a4-907f-5e2e5ad6193b.png)

**Comment by note286** [comment, marked as answer]
正常写就可以。

```latex
\documentclass{xdupgthesis}
\begin{document}
\chapter{title}
测试测试测试测试测试测试测试测试测试测试测试测试测试
\begin{equation}
a+b=c
\end{equation}
测试测试测试测试测试测试测试测试测试测试测试测试测试

测试测试测试测试测试测试测试测试测试测试测试测试测试
\begin{equation}
a+b=c
\end{equation}

测试测试测试测试测试测试测试测试测试测试测试测试测试
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/224328498-97c9437d-041c-48bb-8c5b-11b64bdc1805.png)

**Reply by eutopia-61** [reply]
我在一个定义里面使用了公式，感觉看着不太像空了两行
```
\begin{definition}[泊松分布]
	如果一个随机变量X的概率函数满足：
	\begin{equation}
		\label{equ:2-1}
		P\left( X=k\right) = 
		\frac{\lambda^k}{k!} e^{-\lambda} \quad
		\lambda > 0, k = 0,1,2,...,n
	\end{equation}
\end{definition}
```
![image](https://user-images.githubusercontent.com/67130065/224334959-d5949a91-f3bf-4554-8b89-049043c13e01.png)

**Reply by note286** [reply]
视觉误差，你仔细测量一下。

**Reply by eutopia-61** [reply]
所以是以那个P那个字母作为参考，上面一行，下面一行

**Reply by note286** [reply]
是的。

**Reply by eutopia-61** [reply]
好的，感谢
